src/adagrams.js

```diff
@@ -1,8 +1,86 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const letterPool = { A: 9, B: 2, C: 2, D: 4, E: 12, F: 2, G: 3, H: 2, I: 9, 
+                        J: 1, K: 1, L: 4, M: 2, N: 6, O: 8, P: 2, Q: 1, R: 6, 
+                        S: 4, T: 6, U: 4, V: 2, W: 2, X: 1, Y: 2, Z: 1 };
+    const letterArray = [];
+    const drawn = [];
+    
+    for (let key in letterPool){
+      for( let i = 0; i < letterPool[key]; i += 1){
+        letterArray.push(key);
+      }
```

Comment:
I really like that you store the letter frequencies and then build the pool on the fly - this is much easier to read than a giant array full of 9 'A's, 2 'B's, etc.